recipes/libssh2/all/conandata.yml

```diff
@@ -14,13 +17,9 @@ sources:
 patches:
   "1.10.0":
     - patch_file: "patches/0001-threads.patch"
```

Comment:
Would you be able to provide a `patch_description` and `patch_type` field so we can get rid of those warnings? Thx!